v63003/u_inref/gtm8794.csh

```diff
@@ -37,8 +38,14 @@ $MUPIP FREEZE -OFF DEFAULT
 
 echo "# Updating Database"
 # Reassigning jnl file for the database to be in the temp folder (Randomly set to before/nobefore by the tests framework)
-$MUPIP SET -REGION DEFAULT -JOURNAL=disable -REPLICATION=off>&jnl.outx
-$MUPIP SET -REGION DEFAULT $tst_jnl_str -REPLICATION=on>&jnl.outx
+$MUPIP SET -REGION DEFAULT -JOURNAL=disable -REPLICATION=OFF>&jnl.outx
+if($status) then
+	echo "Bad MUPIP SET"
```

Comment:
MUPIP SET -> MUPIP SET JOURNAL

Line 43 and 47 currently have same error message so one will have no clue which line had the error.